conf/UsbInvocationScript.txt

```diff
@@ -0,0 +1,2 @@
+usb_disable_secure_boot noreset;
+usb_one_time_boot usb nolog;
```

Comment:
Could you please check whether UsbInvocationScript.txt allows comments at all so we can provide breadcrumbs in the file itself about where this came from?

Reply:
This file was generated by their custom tool, and its format does not appear to be documented anywhere. Indeed, one of my replies to Jason was to request the complete syntax/grammar of these files. Thus, I think it would be an undue risk to add comments directly in the file. I can, however, add a small blurb to the target documentation that I've drafted, so these options are documented within the EVE source tree.

Reply:
Fair enough. Since you'll be moving this file to pkg/mkimage-raw-efi you can provide ample comments in Dockerfile there